### PR TITLE
Use manageiq/manageiq-pods images

### PR DESCRIPTION
**Pull request by simaishi**
Use manageiq/manageiq-pods images built in Docker Hub.

All 3 images come from 'manageiq-pods' and each image will have different tag which contains the name of the image ('app', 'memcached' or 'postgresql').

### CODE REVIEW

**Comment by simaishi** [comment]
@fbladilo please review - tested in Origin environment.

File: templates/miq-template.yaml [hunk 1]
```diff
@@ -272,7 +272,7 @@ objects:
     annotations:
       description: "Keeps track of changes in the ManageIQ postgresql image"
   spec:
-    dockerImageRepository: docker.io/fbladilo/miq-postgresql
+    dockerImageRepository: docker.io/manageiq/manageiq-pods
```
Comment (fbladilo) [inline templates/miq-template.yaml]: @simaishi I don't think we need to keep track of three different ImageStreams pointing to the same docker repository. We could should use a single "miq-images" IS and reference that across the template. We moved away from three different repos and we now call the specific images for pods by tags, not by image names anymore.

Comment (simaishi) [inline templates/miq-template.yaml]: Oh, that's true.  IS itself isn't tied to any image.  Will update - thanks!

Comment (fbladilo) [inline templates/miq-template.yaml]: @simaishi Correct for instance : 

```
from:
          kind: ImageStreamTag
          name: miq-images:postgresql-latest

- memcached
        from:
          kind: ImageStreamTag
          name: miq-images:memcached-latest
```

**Comment by fbladilo** [comment]
@simaishi LGTM :+1:

**Comment by chessbyte** [comment]
@simaishi Why is this labeled `euwe/conflict`?

**Comment by simaishi** [comment]
I'd need to change some values in this PR to make it work for Euwe.  Will create a separate PR.

**Comment by simaishi** [comment]
Backported to Euwe via https://github.com/ManageIQ/manageiq-pods/pull/75

**Pull request merged**